eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,6 +1,15 @@
 variables:
   OfficialBuildId: $(Build.BuildNumber)
   skipComponentGovernanceDetection: true
-  PipelineArtifactName: packages
   nugetMultiFeedWarnLevel: 'none'
   GDN_SUPPRESS_FORKED_BUILD_WARNING: true
+
+  # Manual builds of main or hotfix/* are "release builds" and get no dynamic version suffix.
+  ${{ if and(or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/heads/hotfix/')), eq(variables['Build.Reason'], 'Manual')) }}:
```

Comment:
I'm still not a fan of hard-coding these branch patterns. Why not do this for all manual builds? They won't release unless approved anyway.

Reply:
reverted